src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -6,50 +6,40 @@
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
-using Common.Logging;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
 using Microsoft.CodeAnalysis.Text;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.DotNet.ProjectModel;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyModel;
 using Microsoft.Extensions.Logging;
 using OmniSharp.Models.v1;
-using OmniSharp.ScriptCs.Extensions;
 using OmniSharp.Services;
-using ScriptCs;
-using ScriptCs.Contracts;
-using ScriptCs.Hosting;
-using LogLevel = ScriptCs.Contracts.LogLevel;
 
-namespace OmniSharp.ScriptCs
+namespace OmniSharp.Script
 {
     [Export(typeof(IProjectSystem))]
-    public class ScriptCsProjectSystem : IProjectSystem
+    public class ScriptProjectSystem : IProjectSystem
     {
-        private static readonly string BaseAssemblyPath = Path.GetDirectoryName(typeof(object).Assembly.Location);
-        CSharpParseOptions CsxParseOptions { get; } = new CSharpParseOptions(LanguageVersion.CSharp6, DocumentationMode.Parse, SourceCodeKind.Script);
-        IEnumerable<MetadataReference> DotNetBaseReferences { get; } = new[]
-            {
-                MetadataReference.CreateFromFile(typeof(object).GetTypeInfo().Assembly.Location),        // mscorlib
-                MetadataReference.CreateFromFile(typeof(Enumerable).GetTypeInfo().Assembly.Location),    // systemCore
-                MetadataReference.CreateFromFile(typeof(IScriptHost).Assembly.Location)                  // scriptcsContracts
-            };
-
-        OmnisharpWorkspace Workspace { get; }
-        IOmnisharpEnvironment Env { get; }
-        ScriptCsContext Context { get; }
-        ILogger Logger { get; }
+        private CSharpParseOptions CsxParseOptions { get; } = new CSharpParseOptions(LanguageVersion.CSharp6, DocumentationMode.Parse, SourceCodeKind.Script);
```

Comment:
Use `LanguageVersion.Default` to ensure that the latest version of the language is used?